inlineplz/linters/__init__.py

```diff
@@ -277,13 +277,20 @@ def run_command(command, log_on_fail=False, log_all=False):
     shell = False
     if os.name == 'nt':
         shell = True
+    local_env = {}
+    safe_envvars = [
+        b'PATH', b'PYTHONPATH', b'GOPATH', b'SystemRoot',
```

Comment:
Yeah, that's a really good point about the per language ballooning. Running the comment portion in a separate container would be an option, but then you'd need to think about authorization to be able to use the comment service... how deep would that rabbit hole go?